tests/fixtures/issueData/ipfs-roadmap-98.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
obtained via the code in https://github.com/pln-planning-tools/StarMaps/pull/222